EIPS/eip-nft_multi_user.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
```suggestion
---
eip: 7507
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
{
    "id": 1337
    "jsonrpc": "2.0",
    "error": {
        "code": -32003,
        "message": "Transaction rejected"
    }
}